### ISSUE TITLE
[QUESTION] InputLeftElement/InputRightElement with an IconButton

### ISSUE THREAD

**Issue by DevLexicon** [kgnugur/formik-chakra-ui#81]
Hello,

I wanted to first start off by saying thank you for creating this wonderful and great package 🙌🏻

I was wondering if there is a way to implement Chakra UI's [InputLeftElement/InputRightElement](https://chakra-ui.com/docs/form/input#left-and-right-addons) with an [IconButton](https://chakra-ui.com/docs/form/icon-button) within your [InputControl](https://happy-lovelace-6f257c.netlify.app/index.html#inputcontrolprops) 🤔

Basically I would love to recreate this [Chakra Pro: Login with centred form (Password Field)](https://pro.chakra-ui.com/components/application/authentication) component to work with Formik!

Thank,
Donald Louch, CEO and Founder of DevLexicon

**Comment by kgnugur** [comment]
Sorry for the delay,

Well, we now export `FormControl` so you can create your custom components using `FormControl`.

Example:
`            <FormControl
              name="customField"
              label="Custom FormControl"
              helperText="Helper text"
            >
              <Field
                as={Input}
                name="customField"
                placeholder="A custom field"
              />
            </FormControl>
`
